gensim/models/coherencemodel.py

```diff
@@ -113,16 +113,18 @@ class CoherenceModel(interfaces.TransformationABC):
     def __init__(self, model=None, topics=None, texts=None, corpus=None, dictionary=None,
                  window_size=None, coherence='c_v', topn=10, processes=-1):
         """
-        Args:
```

Comment:
@prerna135 Please look at the [example](http://www.sphinx-doc.org/en/stable/ext/example_google.html) and check that plugin works correctly ([napoleon ext for sphinx](https://github.com/RaRe-Technologies/gensim/blob/a3965e9b50cb358e778d0565367029bf3d406a7c/docs/src/conf.py#L27))